### PR TITLE
README: Update with lots of usage examples

### PR DESCRIPTION
**Pull request by ollieparanoid**
### Changes:
* renamed "supported devices" to "devices"
* fix coverage button to URL suggested by coveralls.io
* link to postmarketOS.org
* lots of usage examples
* use `pmbootstrap` alias

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,50 +12,139 @@ For in-depth information please refer to the [postmarketOS wiki](https://wiki.po
...
+Please refer to the [postmarketOS wiki](https://wiki.postmarketos.org) for in-depth coverage of topics such as [porting to a new device](https://wiki.postmarketos.org/wiki/Porting_to_a_new_device) or [installation](https://wiki.postmarketos.org/wiki/Installation_guide). The help output (`pmbootstrap -h`) has detailed usage instructions for every command. Read on for some generic examples of what can be done with `pmbootstrap`.
 
-```shell
+### Basics
+Initial setup:
+```
 $ git clone https://github.com/postmarketOS/pmbootstrap
 $ cd pmbootstrap
-$ ./pmbootstrap.py init
+$ alias pmbootstrap=$PWD/pmbootstrap.py
```
Comment (magmastonealex) [inline README.md]: This looks like a great resource for people new to the project!


Maybe intentional, but perhaps mention that this line could go in ~/.profile or ~/.bashrc to ensure it persists across sessions?

Comment (z3ntu) [reply README.md]: I have personally aliased this to `pmb` as it's more concise

Comment (drebrez) [reply README.md]: @z3ntu it's better with `pmbootstrap` since everywhere in the wiki the example commands are written with `pmbootstrap ...`

Comment (eMPee584) [reply README.md]: how about a symlink in `/usr/local/bin` ?

Comment (z3ntu) [reply README.md]: @eMPee584 Not every distro has /usr/local/bin in their PATH by default.

Comment (ollieparanoid) [reply README.md]: Good point, I've updated the instructions. It links to the part in the porting guide now, where we explain  how to set up the shortcut (someone could also add a note how to do it with a shell alias there).

**Comment by ollieparanoid** [comment]
Thanks for the comments!
I've updated it with even more examples and a link to [Porting guide#Shortcut](https://wiki.postmarketos.org/wiki/Porting_to_a_new_device#Shortcut). Thoughts?

File: README.md [hunk 2]
```diff
@@ -15,50 +12,171 @@ For in-depth information please refer to the [postmarketOS wiki](https://wiki.po
...
+```
 
-### Testing
+Safely delete all chroots:
+```
+$ pmbootstrap zap
+```
 
+### Device Porting Assistance
+Analyze Android [`boot.img`](https://wiki.postmarketos.org/wiki/Glossary#boot.img) files:
```
Comment (MartijnBraam) [inline README.md]: Should we mention here that it can also be a recovery image?

Comment (ollieparanoid) [reply README.md]: good idea!

**Pull request merged**